index.html

```diff
@@ -1 +1,26 @@
-
+<!doctype html>
+<html>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width,initial-scale=1" />
+		<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css" />
```

Comment:
What is this?

Reply:
Adding W3.CSS

Reply:
no w3.css

Reply:
`not in my opinion`

Reply:
`
do the stupid
`

Reply:
Well W3.css makes it look good on all devices you know.

Reply:
### Why I don't

> I will not be forced

Reply:
OK no w3.css.

Reply:
- [ ] make it w3-css
- [ ] only do `Pull requests`